js/modules/k6/grpc/client_test.go

```diff
@@ -87,7 +91,19 @@ func TestClient(t *testing.T) {
 		},
 	}
 
+	cwd, err := os.Getwd()
+	require.NoError(t, err)
+
+	initEnv := &common.InitEnvironment{
+		Logger: logrus.New(),
+		CWD:    &url.URL{Path: cwd},
+		FileSystems: map[string]afero.Fs{
+			"file": afero.NewOsFs(),
```

Comment:
See https://github.com/loadimpact/k6/blob/c51095ad7304bdd1e82cdb33c91abc331533b886/loader/filesystems.go#L38-L43  and https://github.com/loadimpact/k6/blob/c51095ad7304bdd1e82cdb33c91abc331533b886/js/bundle_test.go#L621-L632  for how to fix it on windows , I think :thinking: